### ISSUE TITLE
Forking

### ISSUE THREAD

**Issue by kapaud** [ibm-developer-skills-network/gkpbt-css-circle#10108]
I am getting the following message when trying to fork

No available destinations to fork this repository.

    [kapaud/gkpbt-css-circle](https://github.com/kapaud/gkpbt-css-circle)

I am not able to fork.  Does anybody have any idea about this?_

**Comment by tevthe** [comment]
You are trying to fork your own fork of the example project.
"Cannot fork because you own this repository and are not a member of any organizations."

**Comment by ThummalaChakravarthi2** [comment]
This message usually means you've already forked the repository to your GitHub account or to an organization you belong to

**Comment by Usman-52** [comment]
> This message usually means you've already forked the repository to your GitHub account or to an organization you belong to


Yes, I was facing the same issue, but after reading the message, I realized I already had this repo (forked).
